### PR TITLE
Use GA UA and gtags

### PR DESCRIPTION
**Pull request by kagarmoe**
### Description

We're not getting the full range of analytics that I want to see. This tries to add the UA and G tags. 

Based on:
https://developers.google.com/analytics/devguides/collection/ga4/basic-tag?technology=gtagjs
https://github.com/gohugoio/gohugoioTheme/blob/master/layouts/partials/gtag.html

### Issues Resolved

[List any existing issues this PR resolves, or any Discourse or
StackOverflow discussion that's relevant]

### Check List

- [ ] Spell Check
- [ ] Local build
- [ ] Examine the local build
- [ ] All tests pass

### CODE REVIEW

**Review by mjingle — APPROVED** [review]
tenor-186784892

File: config.toml [hunk 1]
```diff
@@ -80,12 +80,14 @@ notAlternative = true
...
   enable_search = true
   robots = ''
+  images = ["chef-progress-logo.svg"]
   menuOrder = ["overview", "automate", "compliance",  "desktop",  "infra", "habitat", "inspec", "workstation", "effortless", "release_notes", "legacy", "extra"]
 
 [taxonomies]
   category = "categories"
   tag = "tags"
   resource = "resources"
+  series = "series"
```
Comment (IanMadd) [inline config.toml]: Do we need these changes?

**Comment by IanMadd** [comment]
One thing that might make things simpler would be to define the two different tags in the config.toml, 

```
googleAnalyticsUA = "UA-183953607-1"
googleAnalyticsG = "G-CH0D7HCR73"
```

And then just call them: `{{ $.Site.Params.googleAnalyticsUA }}` or `{{ $.Site.Params.googleAnalyticsG }}`.

Something like that.

File: themes/docs-new/layouts/partials/gtag.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{{ with .Site.GoogleAnalytics }}
```
Comment (IanMadd) [inline themes/docs-new/layouts/partials/gtag.html]:
```suggestion
{{ with .Site.GoogleAnalytics }}
```
We can ditch the `{{ with .Site.GoogleAnalytics }}`, and also the `{{end}}` at the bottom.

File: config.toml (hunk 1 above)
Comment (kagarmoe) [reply config.toml]: Not in this PR...it is noise from something else I'm working on.

**Comment by IanMadd** [comment]
In the gtag partial there's this bit: `'dimension2': '{{ getenv "BRANCH" }}`. I think Hugo has that in their theme because they're doing deploys from two branches so I think they're tracking data from both. I could be wrong about that, but I don't think we'll get anything out of that. But it also won't hurt.

**Review by IanMadd — APPROVED** [review]

**Comment by kagarmoe** [comment]
I think it might be helpful for A/B testing, although I'm not sure.

**Comment by IanMadd** [comment]
We might want to remove the other part too `'dimension1': '{{ $site }}'`. Hugo uses that theme for two sites so they're tracking both.

File: themes/docs-new/layouts/partials/gtag.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
+<!-- Global site tag (gtag.js) - Google Analytics -->
+<script async src="https://www.googletagmanager.com/gtag/js?id={{ $.Site.Params.googleAnalyticsUA }}"></script>
+<script>
+  window.dataLayer = window.dataLayer || [];
+  function gtag(){dataLayer.push(arguments);}
+  gtag('js', new Date());
+
+  {{ $site := $.Site.BaseURL | replaceRE "^https?://(www\\.)?([^/]+).*" "$2" }}
```
Comment (IanMadd) [inline themes/docs-new/layouts/partials/gtag.html]: We can cut that out too.
```suggestion
```

**Pull request merged**